autoagora/logs_db.py

```diff
@@ -27,9 +51,96 @@ def return_query_body(self, query):
         self, subgraph_ipfs_hash: str, min_count: int = 100
     ):
+        async with self.pgpool.acquire() as connection:
+            rows = await connection.fetch(
+                """
+                SELECT
+                    hash,
+                    query
+                    
+                FROM
+                    query_skeletons
+                INNER JOIN
+                (
+                    SELECT
+                        query_hash as qhash,
+                        count(id) as count_id
+                    FROM
+                        query_logs
+                    WHERE
+                        subgraph = $1
+                        AND query_time_ms IS NULL
+                    GROUP BY
+                        qhash
+                    HAVING
+                        Count(id) >= $2
+                ) as query_logs
+                ON
+                    qhash = hash
+                ORDER BY
+                    count_id DESC
+                """,
+                subgraph_ipfs_hash,
+                min_count,
+            )
+            return [
+                LogsDB.MRQ_Info(
+                    hash=row[0],
+                    query=self.return_query_body(row[1])
+                    if self.return_query_body(row[1])
+                    else "null",
+                )
+                for row in rows
+            ]
+
+    async def get_query_variables_from_query_hash(self, hash):
+        async with self.pgpool.acquire() as connection:
+            query_variables = await connection.fetch(
+                """
+                SELECT
+                    query_variables
+                FROM 
+                    query_logs
+	            WHERE
+                    query_hash = $1;
+                """,
+                hash,
+            )
+            return query_variables
+
+    async def save_generated_aa_query_values(
+        self, query: MRQ_Info, subgraph: str, query_variables
+    ):
+        async with self.pgpool.acquire() as connection:
+            query_variables = await connection.execute(
+                """
+                    INSERT INTO mrq_query_logs (
+                            subgraph,
+                            query_hash,
+                            timestamp,
+                            query_time_ms,
+                            query_variables
+                        )
+                        VALUES (%s, %s, %s, %s, %s)
+                    """,
+                (
+                    subgraph,
+                    query.hash,
+                    query.timestamp,
+                    query.query_time_ms,
+                    query_variables,
+                ),
+            )
+
+    async def get_most_frequent_queries(
+        self, subgraph_ipfs_hash: str, min_count: int = 100, mrq_table: bool = False
+    ):
+        most_frequent_query_table = "query_logs"
+        if mrq_table:
+            most_frequent_query_table = "mrq_query_logs"
```

Comment:
Adding a new line for readability

```suggestion
            most_frequent_query_table = "mrq_query_logs"
            
```